### DISCUSSION TITLE
Github Copilot chat stop working.

### DISCUSSION THREAD

**Discussion by kgp7812** [community/community#106670] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have signed for paid verision of the Github Copilot.  Initially I was able to use  Github Chat functions, But after few hours it stop working and getting following message when try to run simple prompt.
Error Message:
"Oops,your response got filtered. Vote down if you think this should't have happend."

**Comment by davecheney** [comment, marked as answer]
@kgp7812 i'm sorry you're experiencing difficulties. Could you do one thing for me; please make sure you're using the latest version of the Copilot extension (which will auto install the corresponding chat extension). If you haven't updated VScode this month, you'll need to do that first as newer versions of the extension are gated behind newer versions of the editor.

**Comment by mmohindra** [comment]
What is the most stable version of VSCOde _ Github copilot plugin to install on windows 10 ? I cannot get any version to work atleast the chat part

**Comment by akathedeveloper** [comment]
Right now which version you are using can you tell please?

**Comment by mmohindra** [comment]
VSCode 1.89, copilot 1.95 and chat 1.15
________________________________
